geonode/settings.py

```diff
@@ -334,7 +334,9 @@
 CACHES = {
     # DUMMY CACHE FOR DEVELOPMENT
     "default": {
-        "BACKEND": "django.core.cache.backends.dummy.DummyCache",
```

Comment:
I guess you removed the `BACKEND` key here erroneously

Reply:
Oh yes! Thanks @afabiani